.env.vercel

```
@@ -0,0 +1,2 @@
+# Created by Vercel CLI
+VERCEL_OIDC_TOKEN="eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6Im1yay00MzAyZWMxYjY3MGY0OGE5OGFkNjFkYWRlNGEyM2JlNyJ9.eyJpc3MiOiJodHRwczovL29pZGMudmVyY2VsLmNvbS9tYXhpbS1hLXMtcHJvamVjdHMiLCJzdWIiOiJvd25lcjptYXhpbS1hLXMtcHJvamVjdHM6cHJvamVjdDp5b211OmVudmlyb25tZW50OmRldmVsb3BtZW50Iiwic2NvcGUiOiJvd25lcjptYXhpbS1hLXMtcHJvamVjdHM6cHJvamVjdDp5b211OmVudmlyb25tZW50OmRldmVsb3BtZW50IiwiYXVkIjoiaHR0cHM6Ly92ZXJjZWwuY29tL21heGltLWEtcy1wcm9qZWN0cyIsIm93bmVyIjoibWF4aW0tYS1zLXByb2plY3RzIiwib3duZXJfaWQiOiJ0ZWFtX3VVWTdXVXNLNFFSeGhFTEZZT1hHYXF1cCIsInByb2plY3QiOiJ5b211IiwicHJvamVjdF9pZCI6InByal94WUJYaEdWSGU2aWp1MVRmQnFGMm5UVGY4YVoxIiwiZW52aXJvbm1lbnQiOiJkZXZlbG9wbWVudCIsInBsYW4iOiJob2JieSIsInVzZXJfaWQiOiJ0YWhEYUZGTDlBNHY4ZGk1VjhRU1htckYiLCJjbGllbnRfaWQiOiJjbF9IWXlPUEJOdEZNZkhoYVVuOUw0UVBmVFp6NlRQNDdicCIsIm5iZiI6MTc3OTMxMjMwNCwiaWF0IjoxNzc5MzEyMzA0LCJleHAiOjE3NzkzNTU1MDR9.in8ihbsp6QadDUCrkJxcFM7M8f9Pbajl0VejVgcV29P0uS-bFM9YQVHJlKXX5vTuOTArwjhopB1-T31l15ay4kJovYHBOkFgGjCknlwTCauTwXyVy0YJ4QYPMDyGhoQ0z1DdgcnDExveAMp9cE4I7UJOgAHRWCEi6LSh-c7kw_0hZLJtjCVFkQ9MmCT48EiH9tz8seIYAY1Oq0dw9NL-6_TwFwirfTgHnhoo_8_8gkjYGLqqV25aGyrvTjoRjwIdFtKsWmp5jV6qkJvKQD7zP0eRTuG_MKHAUCZDE2ECxwPy8IDX9c2BfDCP133eCOVlm8AlPgJhDyOeywJF7pB-9g"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)